tools/check

```diff
@@ -241,6 +241,19 @@ should_run_build_runner() {
     return 1
 }
 
+run_l10n() {
+    files_check lib/generated/l10n/'*'.dart \
```

Comment:
nit: like

    local schema_dir=test/model/schemas/

we can extract a local variable for the output path